{
  "code": -32003,
  "data": {
    "code": 10,
    "message": "Assert Exception",
    "name": "assert_exception",
    "stack": [
      {
        "context": {
          "file": "database_api.cpp",
          "hostname": "",
          "level": "error",
          "line": 1269,
          "method": "list_votes",
          "timestamp": "2020-09-29T17:09:19"
        },
        "data": {},
        "format": "false: Unknown or unsupported sort order"
      }
    ]
  },
  "message": "Assert Exception:false: Unknown or unsupported sort order"
}
